documentation/src/main/asciidoc/core/manual/en_US/configuration.adoc

```diff
@@ -250,4 +250,9 @@ For more information refer to the <<customize-dbms-dialect,Customize DBMS dialec
 [[configuration-entity-manager-factory-integrators]]
 === Entity manager factory integrators
 
-A little integration layer to extract the dbms of an entity manager and register `JpqlFunctionGroup` instances with the entity manager. This is normally provided by an integration module for persistence providers.
\ No newline at end of file
+A little integration layer to extract the dbms of an entity manager and register `JpqlFunctionGroup` instances with the entity manager. This is normally provided by an integration module for persistence providers.
+
+[[service-loader-configuration-integrators]]
+=== Extending configuration from integrations
+
+Integrations may provide additional configuration to the bootstrapping `CriteriaBuilderConfiguration` through a `CriteriaBuilderConfigurationContributor` Service Provider.
```

Comment:
It would be great if you could list some concrete use cases like e.g. contributing functions etc.

Reply:
Added more documentation.